### ISSUE TITLE
single and double quotes not properly unescaped

### ISSUE THREAD

**Issue by lipp** [zeit/styled-jsx#96]
styles containing quotes don't work:

```jsx
<style jsx global>{`
  body {
    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
    }
`}</style>

```

gets inserted in browser as:

```css
body {
  font-family: font-family: -apple-system, BlinkMacSystemFont, &#x27;Segoe UI&#x27;, Helvetica, Arial, sans-serif, &#x27;Apple Color Emoji&#x27;, &#x27;Segoe UI Emoji&#x27;, &#x27;Segoe UI Symbol&#x27;
}
```

**Comment by HaNdTriX** [comment]
Just wanted to confirm @lipp s report:

Doesn't work:

	li:after {
	  content: "\f178";
	  font-family: FontAwesome;
	}

Hack to make it work: 


	li:after {
	  content: "\\f178";
	  font-family: FontAwesome;
	}

**Comment by giuseppeg** [comment]
should be fixed by #134
